Library/Homebrew/cask/lib/hbc/artifact/pkg.rb

```diff
@@ -2,7 +2,7 @@
 
 require "hbc/utils/hash_validator"
 
-require "vendor/plist/plist"
```

Comment:
I think it’s better to use the full require part for these as you can quickly search for which files use vendor files.